test/routes/src/server/routes/t_github.js

```diff
@@ -640,12 +641,21 @@ const hasStatus = (expected) => {
   };
 };
 
-const hasMessage = (expected) => {
+const hasPayloadMessage = (expected) => {
```

Comment:
Why these are called `hasPayloadMessage` `hasBodyMessage` when they seem to be checking `code` property?